src/components/Paragraph/README.md

```diff
@@ -1,61 +1,105 @@
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-### Small (Fine Print) Paragraph
+Use a **small paragraph** for fine print, or details that are secondary to the main content.
 
 ```jsx
-<Paragraph size="small">
+<Paragraph small>
 	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
 	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-## Colors
+Use an **inverse paragraph** if you need to put a paragraph on top of a dark background.
+
+```jsx
+<Paragraph inverse>
+	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
+	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
+	lectus in sem viverra volutpat.
+</Paragraph>
+```
 
-### Paragraph with custom color
+For both inverse and default type colours, you can also use a dark and light variant. Use sparingly when needed to provide an additional level of hierarchy or emphasis.
 
 ```jsx
-<Paragraph color="accent">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="medium">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="light">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="danger">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="success">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
+<Paragraph light>
+	Light
```

Comment:
Some mad tabs going on in here. You using Atom? 😆

Reply:
SURE AM. (_!_)